atlassian/bitbucket/base.py

```diff
@@ -1,8 +1,12 @@
 # coding=utf-8
 
+from datetime import datetime
 from ..rest_client import AtlassianRestAPI
 
 
+CONF_TIMEFORMAT = "%Y-%m-%dT%H:%M:%S.%f%z"
+
+
```

Comment:
Move as attribute of `BitbucketBase`, than it can be accessed with every derived class or instance without an additional import.

Reply:
Done.